src/commands/jobs/util.ts

```diff
@@ -286,3 +286,27 @@ export const itemOutputFormat = {
 		'transferred_on'
 	]
 };
+
+export const securityMappingOutputFormat = {
+	table: [
+		{
+			header: 'Source',
+			property: 'source',
+			transform: val => val && (val.email || val.username || val.id)
+		},
+		{
+			header: 'Destination',
+			property: 'destination',
+			transform: val => val && (val.email || val.username || val.id)
+		},
+		{
+			header: 'Resolution',
+			property: 'mapper'
```

Comment:
@avolinsky mentioned that he will be changing the `mapper` property name. I'll update it here as soon as his change is made.